inc/class-wpseo-admin-bar-menu.php

```diff
@@ -268,7 +268,7 @@ protected function add_analysis_submenu( WP_Admin_Bar $wp_admin_bar ) {
 			'parent' => self::ANALYSIS_SUBMENU_IDENTIFIER,
 			'id'     => 'wpseo-inlinks-ose',
 			'title'  => __( 'Check Inlinks (OSE)', 'wordpress-seo' ),
```

Comment:
I guess also the link text should be changed as it's not OSE (Open Site Explorer) any longer: the new service's name is Link Explorer so still referring to `OSE` could be confusing. I'd also update the ID.